dunwoody_disaster/views/characterState.py

```diff
@@ -0,0 +1,65 @@
+from dunwoody_disaster.CharacterFactory import Character
+import dunwoody_disaster as DD
+from PySide6.QtWidgets import (
+    QWidget,
+    QGridLayout,
+    QLayout,
+    QLabel,
+)
+from PySide6.QtGui import QPixmap
+from PySide6.QtCore import Qt
+
+
+class CharacterState(QWidget):
+    def __init__(self, character: Character):
+        super().__init__()
+        self.character = character
+
+        self.update_ui()
+        return
+
+    def update_ui(self):
+        self.setLayout(self.create_layout())
```

Comment:
```suggestion
        self.setLayout(self.create_layout())
        return
```
Remove pointless function

Reply:
Or maybe change the update_ui button to update the health, magic, and mechanical values?